xdsl/parser/core.py

```diff
@@ -515,7 +515,7 @@ def parse_optional_argument(self, expect_type: bool = True) -> Argument | None:
             type = self.parse_type()
         return self.Argument(name_token.span, type)
 
-    def parse_argument(self, expect_type: bool = True) -> Argument:
+    def parse_argument(self, *, expect_type: bool = True) -> Argument:
```

Comment:
minor API change, this only causes new pyright errors, and shouldn't break existing code

Reply:
Ah no I was wrong this will break code...

Reply:
I can revert it or we can keep it, I would rather have the argument label be mandatory, but I don't want to force this in.